include/zephyr/dt-bindings/battery/battery.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 Embeint Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The following open-circuit voltage curves have been extracted from the datasheets of the
+ * listed battery parts. They will not be 100% correct for all batteries of the chemistry,
+ * but should provide a good baseline. These curves will also be affected by ambient temperature
+ * and discharge current.
+ *
+ * Each curve is 11 elements representing the OCV voltage in microvolts for each charge percentage
+ * from 0% to 100% inclusive in 10% increments.
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_BATTERY_BATTERY_H_
```

Comment:
I am a bit worried that this list would be difficult to maintain, as people will want to have their own batteries there.

Do you think it is possible to define those curves in the device tree instead? This is an approach that has been followed for the LUT table of epaper displays, see for instance https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/shields/waveshare_epaper/waveshare_epaper_gdeh029a1.overlay#L33

Reply:
These are just default curves meant to simplify the DTS for cases where absolute accuracy is not required.
You can choose to provide your own curve if you wish:
```
ocv-capacity-table-0 = <3000000 3100000 3200000 ...>;
```

Reply:
Great, thanks for the explanation